addons/im_livechat/static/src/core/web/thread_model_patch.js

```diff
@@ -26,6 +27,15 @@ patch(Thread.prototype, {
         } else if (status === "need_help") {
             return _t("Looking for help");
         }
-        return "";
+        return "In progress";
+    },
+    updateLivechatStatus(livechat_status) {
```

Comment:
:pick: or at least don't use snake case :snake: 
```suggestion
    updateLivechatStatus(status) {
```

Reply:
> or at least don't use snake case 🐍

I get JS ecosystem is camelCase but python ecosystem is snake_case. I really think we should relax this and sometimes just go with the one that is less friction in the code.
Note a big deal here but in many circumstances the less code friction in Odoo JS is frequently snake_case.